### PR TITLE
Fix/clean up todos

### PR DESCRIPTION
**Pull request by ioanaroceanu2000**

### CODE REVIEW

File: exchanges/vamm/src/libraries/vamm-utils/VammHelpers.sol [hunk 1]
```diff
@@ -10,6 +10,8 @@ import { SafeCastU256, SafeCastI256 } from "@voltz-protocol/util-contracts/src/h
 
 import { UD60x18, ZERO, ud, convert } from "@prb/math/UD60x18.sol";
 
+import { UD60x18, ZERO, ud, convert } from "@prb/math/UD60x18.sol";
```
Comment (0xZenus) [inline exchanges/vamm/src/libraries/vamm-utils/VammHelpers.sol:13]: this is duplicated.

File: exchanges/vamm/src/modules/PoolModule.sol [hunk 2]
```diff
@@ -126,8 +127,9 @@ contract PoolModule is IPoolModule {
 
             vamm.executeDatedMakerOrder(accountId, position.tickLower, position.tickUpper, -position.liquidity.toInt());
 
-            // todo: shouldn't we convert liquidity to base here?
-            closedUnfilledBasePool += position.liquidity.toInt();
+            (uint256 absClosedBase,) = amountsFromLiquidity(position.liquidity, position.tickLower, position.tickUpper);
+
+            closedUnfilledBasePool += absClosedBase.toInt();
```
Comment (0xZenus) [inline exchanges/vamm/src/modules/PoolModule.sol:132]: Could we make the return of this function to uint256?

File: exchanges/vamm/src/libraries/vamm-utils/VammHelpers.sol (hunk 1 above)
Comment (ioanaroceanu2000) [reply exchanges/vamm/src/libraries/vamm-utils/VammHelpers.sol]: thanks! fixed in [6bfe08d](https://github.com/Voltz-Protocol/v2-core/pull/192/commits/6bfe08d3db3c8c6457e97deada8dd6ebccdc2e28)

File: exchanges/vamm/src/modules/PoolModule.sol (hunk 2 above)
Comment (ioanaroceanu2000) [reply exchanges/vamm/src/modules/PoolModule.sol]: yes, makes more sense. changed in [6bfe08d](https://github.com/Voltz-Protocol/v2-core/pull/192/commits/6bfe08d3db3c8c6457e97deada8dd6ebccdc2e28)

**Review by 0xZenus — APPROVED** [review]

**Pull request merged**